googletrans/models.py

```diff
@@ -39,7 +39,7 @@ def __init__(
         origin,
         text,
         pronunciation,
-        parts: List[TranslatedPart] | None,
```

Comment:
you can do this: `parts: List[TranslatedPart] or None`

Reply:
> you can do this: `parts: List[TranslatedPart] or None`

pylance flags it `Binary operator not allowed in type annotation`